GOVERNANCE.md

```diff
@@ -183,6 +183,34 @@ Collaborators might overlook someone with valuable contributions. In that case,
 the contributor may open an issue or contact a collaborator to request a
 nomination.
 
+#### How to review a collaborator nomination
+
+A collaborator nomination can be reviewed in the same way one would review a PR
+adding a feature:
+
+* If you see the nomination as something positive to the project, say so!
+* If you are neutral, or feel you don't know enough to have an informed opinion,
+  it's certainly OK to not interact with the nomination.
+* If you think the nomination can be detrimental to the project, share your
```

Comment:
In cases, I don't find the failed nomination to be detrimental. It just needs more supporting evidence.

Given that the three list points are equivalent to +1, 0, -1 respectively, I would like it to explicitly mention that "needs more positive evidence" can be a case for -1.

```suggestion
* If you think the nomination needs more position evident, or can be detrimental to the project, share your
```

Reply:
I'm not sure requiring the person suggesting the nomination for "position evidence" (besides the list of contributions/interractions) is a reasonable ask – and maybe it's not you mean, but I'd prefer we find a phrasing that makes it clear